docs/api/stories.md

```diff
@@ -2,516 +2,10 @@
-      with icon
-    </Badge>
-  </Story>
-</Preview>
-```
-
-And here's how that gets rendered in Storybook:
-
-<div style="background-color:#F8FAFC">
-TODO: VET THIS IMAGE.
-</div>
-
-<center>
-  <img src="https://raw.githubusercontent.com/storybookjs/storybook/master/addons/docs/docs/media/mdx-page.png" width="100%" />
-</center>
-
-## Embedding stories
-
-Suppose you have an existing story and want to embed it into your docs. Here's how to show a story with ID `some--id` (check the browser URL in Storybook v5+ to see a story's ID):
-
-```js
-import { Story } from "@storybook/addon-docs/blocks";
-
-# Some header
-
-And Markdown here
-
-<Story id="some--id" />
-```
-
-You can also use the rest of the MDX features in conjunction with embedding. That includes source, preview, and prop tables.
-
-## Decorators and parameters
-
-To add [decorators](../writing-docs/mdx#decorators-and-parameters) and [parameters](../writing-docs/mdx#decorators-and-parameters) in MDX:
-
-```js
-<Meta
-  title='MyComponent'
-  decorators={[ ... ]}
-  parameters={{ ... }}
-/>
-
-<Story name="story" decorators={[ ... ]} parameters={{ ... }} >
-...
-</Story>
-```
-
-In addition, global decorators work just like before, e.g. adding the following to your [`.storybook/preview.js`](../configure/overview#configure-story-rendering):
-
-```js
-// .storybook/preview.js
-import { addDecorator, addParameters } from '@storybook/react';
-
-addDecorator(...);
-addParameters({ ... });
-```
-
-#### Documentation-only MDX
-
-Typically, when you use Storybook MDX, you define stories in the MDX documentation is automatically associated with those stories. But what if you want to write Markdown-style documentation and have it show up in your Storybook?
-
-If you don't define stories in your MDX, you can write MDX documentation and associate it with an existing story, or embed that MDX as its own documentation node in your Storybook's navigation.
-
-If you don't define a `Meta`, you can write Markdown and associate with an existing story. See ["CSF Stories with MDX Docs"](../writing-docs/mdx).
-
-To get a "documentation-only story", in your UI, define a `<Meta>` as you normally would, but don't define any stories. It will show up in your UI as a documentation node:
-
-<div style="background-color:#F8FAFC">
-TODO: VET THIS IMAGE.
-</div>
-
-<center>
-  <img src="https://raw.githubusercontent.com/storybookjs/storybook/master/addons/docs/docs/media/mdx-documentation-only.png" width="100%" />
-</center>
-
-## MDX file names
-
-Unless you use a custom webpack configuration, all of your `MDX` files should have the suffix `*.stories.mdx`. This tells Storybook to apply its special processing to the `<Meta>` and `<Story>` elements in the file.
-
-Be sure to update your Storybook config file to load `.stories.mdx` stories, as per the [`addon-docs` installation instructions](https://github.com/storybookjs/storybook/tree/master/addons/docs#installation).
-
-### ArgTypes
```

Comment:
I think this section got dropped by accident.

Reply:
Based on the feedback that i got, there was a need to split it to smaller subpages that target those items. One for mdx, one for csf, one for storiesOf.

@domyen care to chime in

Reply:
Yea, we're splitting these pages up so that we can link to individual pages.
![image](https://user-images.githubusercontent.com/263385/89462381-4e9a5180-d73b-11ea-8458-46484500384d.png)

Reply:
Ignore the StoriesOf API, that's no longer present in the docs